packages/directory/addon/controllers/directory.js

```diff
@@ -4,9 +4,23 @@
  */
 import Controller from '@ember/controller';
 import { getProperties, get, set, computed } from '@ember/object';
-import dirInfo from '../utils/enums/directories';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+import { A as arr } from '@ember/array';
 
 export default Controller.extend({
+  /**
+   * @property {Ember.Controller} application
+   */
+  application: computed(function() {
+    return getOwner(this).lookup('controller:application');
```

Comment:
does `get(this, 'target.currentPath')` work if you're only fetching the application to get the current path?

Reply:
Yes it does. I didn't know about that property